tools/charts/bookkeeper-operator/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: bookkeeper-operator
```

Comment:
Is this a copy-paste of https://github.com/pravega/bookkeeper-operator/tree/master/charts/bookkeeper-operator ?
No worries that this might cause some problems later -- it seems like changes would need to be maintained in two spots (here and there), no?

May I suggest instead pulling the chart that is built from https://asd-nautilus-jenkins.isus.emc.com/job/bookkeeper-operator/ as it is needed, instead? That way there is only one source-of-truth code-wise (the `bookkeeper-operator` repository)